openshift_tools/monitoring/zagg_sender.py

```diff
@@ -27,6 +27,14 @@
 import os
 import yaml
 
+# Add Hawkular sender backend (Optional)
```

Comment:
@yaacov enabling sending the data is different from the module dependency. I prefer that you pull in always this dependency (and yes sending data should be instead optional, default false).